src/applications/hca/config/chapters/veteranInformation/birthInformation.js

```diff
@@ -48,8 +42,8 @@ export default {
           cityOfBirth,
           stateOfBirth: {
             type: 'string',
-            enum: [...states.map(object => object.value), 'Other'],
-            enumNames: [...states.map(object => object.label), 'Other'],
+            enum: [...states50AndDC.map(object => object.value), 'Other'],
```

Comment:
Should the "Other" value here be set to "FG"?

Reply:
That transformation is being done on our backend.